app/Filament/Resources/Vouchers/Tables/VouchersTable.php

```diff
@@ -0,0 +1,54 @@
+<?php
+
+namespace App\Filament\Resources\Vouchers\Tables;
+
+use Filament\Actions\BulkActionGroup;
+use Filament\Actions\DeleteBulkAction;
+use Filament\Actions\EditAction;
+use Filament\Tables\Columns\TextColumn;
+use Filament\Tables\Table;
+
+class VouchersTable
+{
+    public static function configure(Table $table): Table
+    {
+        return $table
+            ->columns([
+                TextColumn::make('code')
+                    ->searchable(),
+                TextColumn::make('company.name')
+                    ->numeric()
+                    ->sortable(),
+                TextColumn::make('consultant.name')
+                    ->numeric()
+                    ->sortable(),
+                TextColumn::make('user.name')
+                    ->numeric()
+                    ->sortable(),
+                TextColumn::make('status')
+                    ->searchable(),
```

Comment:
when you have a enum casted, remember to use the ->badge() to make it visible.


```suggestion
                TextColumn::make('status')
                    ->badge()
                    ->searchable(),
```